pages/linux/du.md

```diff
@@ -1,23 +1,27 @@
 # du
 
-> Estimate file space usage.
+> Disk usage: estimate and summarize file and folder space usage.
 
-- Get a sum of the total size of a file/folder in human readable units:
+- List the sizes of a folder and any subfolders, in the given unit (B/KB/MB):
 
-`du -sh {{file_or_directory}}`
+`du -{{b|k|m}} {{path/to/folder}}`
 
-- List file sizes of a directory and any subdirectories in KB:
+- List the sizes of a folder and any subfolders, in human-readable form (i.e. auto-selecting the appropriate unit for each size):
 
-`du -k {{file_or_directory}}`
+`du -h {{path/to/folder}}`
 
-- List file sizes of a directory and any subdirectories in MB:
+- Show the size of a single folder, in human readable units:
 
-`du -m {{file_or_directory}}`
+`du -sh {{path/to/folder}}`
 
-- Get recursively, individual file/folder sizes in human readable form:
+- List the human-readable sizes of a folder and of all the files and folders within it (recursively):
```

Comment:
I would say that "folder and of all the files and folders within it" already means "everything" inside it. Following your line of reasoning, since recursively doesn't "need" to be a part of the sentence, I would say it doesn't need to be there at all. Since the meaning is already captured. Just my 2 cents. Let me know what you think.